index.html

```diff
@@ -2,12 +2,21 @@
 <html lang="de">
     <head>
         <meta charset="UTF-8" />
-        <link rel="icon" type="image/png" href="/tree.png" />
         <meta name="viewport" content="width=device-width, initial-scale=1.0" />
         <title>Green Ecolution | Smartes Grünflächenmanagement</title>
         <meta name="author" content="Hochschule Flensburg – Forschungsprojekt">
         <meta name="description" content="Die Projektwebsite zum Forschungsprojekt »Green Ecolution« für smartes Grünflächenmanagement der HS Flensburg.">
-        <meta name="theme-color" content="#659858">
+        <link rel="icon" type="image/svg+xml" href="/public/assets/favicons/favicon.svg" sizes="any">
+        <link rel="apple-touch-icon" sizes="180x180" href="/public/assets/favicons/apple-touch-icon.png">
+        <link rel="icon" type="image/png" sizes="16x16" href="/public/assets/favicons/favicon-16x16.png">
+        <link rel="icon" type="image/png" sizes="32x32" href="/public/assets/favicons/favicon-32x32.png">
+        <link rel="icon" type="image/png" sizes="48x48" href="/public/assets/favicons/favicon-48x48.png">
+        <link rel="icon" type="image/png" sizes="192x192"  href="/public/assets/favicons/android-chrome-192x192.png">
+        <link rel="icon" type="image/png" sizes="144x144" href="/public/assets/favicons/android-chrome-144x144.png">
+        <link rel="icon" type="image/png" sizes="96x96" href="/public/assets/favicons/android-chrome-96x96.png">
```

Comment:
Please remove `/public`

```suggestion
        <link rel="icon" type="image/svg+xml" href="/assets/favicons/favicon.svg" sizes="any">
        <link rel="apple-touch-icon" sizes="180x180" href="/assets/favicons/apple-touch-icon.png">
        <link rel="icon" type="image/png" sizes="16x16" href="/assets/favicons/favicon-16x16.png">
        <link rel="icon" type="image/png" sizes="32x32" href="/assets/favicons/favicon-32x32.png">
        <link rel="icon" type="image/png" sizes="48x48" href="/assets/favicons/favicon-48x48.png">
        <link rel="icon" type="image/png" sizes="192x192"  href="/assets/favicons/android-chrome-192x192.png">
        <link rel="icon" type="image/png" sizes="144x144" href="/assets/favicons/android-chrome-144x144.png">
        <link rel="icon" type="image/png" sizes="96x96" href="/assets/favicons/android-chrome-96x96.png">
```